python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_sobol_multivariate.py

```diff
@@ -0,0 +1,104 @@
+"""
+Estimate Sobol' indices for a function with multivariate output
+===============================================================
+"""
+# %% 
+#
+# In this example, we estimate the Sobol' indices of a function by sampling methods. 
+# This function has several outputs, which leads to the need of aggregated Sobol' indices. 
+#
+
+# %%
+# Introduction
+# ------------
+#
+# In this example we are quantify the sensitivity of inputs to outputs of a function based on Sobol' indices.
+#
+# The function we consider has 5 outputs. In this case, it may be convenient to consider each output separately. It may also be interesting to aggregate the sensitivity indices to that we get a global understanding of the sensitivity of the inputs to the average output. 
```

Comment:
```suggestion
# The function we consider has 5 outputs. In this case, it may be convenient to consider each output separately. It may also be interesting to aggregate the sensitivity indices to get a global understanding of the sensitivity of the inputs to the average output. 
```
I think "inputs" and "average output" are exchanged again. Besides, does it make sense to talk about "average output" in this setting? What if one of the outputs is a pressure and the other a temperature?